packages/core/src/sass/app/_common/components/account-common.scss

```diff
@@ -311,15 +312,32 @@
             border-radius: 0 $BORDER_RADIUS $BORDER_RADIUS 0;
             border-top: 1px solid var(--general-section-1);
             background-color: var(--general-main-1);
+
+            @include desktop-screen {
+                max-width: 68rem;
+            }
+
+            @include mobile-or-tablet-screen {
+                flex-direction: row;
+                justify-content: center;
+                width: 100%;
+                left: 0;
+                align-items: center;
+            }
+
             .back-btn {
-                margin-right: 8px;
+                margin-inline-start: 8px;
 
-                @include mobile-or-tablet-screen {
+                @include tablet-screen {
+                    width: 568px;
```

Comment:
instead of `px` can we use `rem` here as well ? 😬 
Thank you

Reply:
done